webserver/pkgpkr/webservice/recommender_service.py

```diff
@@ -48,25 +48,37 @@ def get_recommendations(self, dependencies):
             # Get average downloads
             res = requests.get(NPM_LAST_MONTH_DOWNLOADS_META_API_URL + f'/{dependency_name}')
             average_downloads = self.get_average_monthly_donwloads(res.json()['downloads'])
-            d['average_downloads'] = average_downloads
+            # commas as thousands separators
+            d['average_downloads'] = f'{average_downloads:,}'
 
-            # Get keywords (i.e. categories)
+            # Get keywords (i.e. categories) and date
             res = requests.get(NPM_DEPENDENCY_META_URL + f'/{dependency_name}')
 
             res_json = res.json()
 
             d['keywords'] = None
+            d['date'] = None
 
             if res_json.get('versions') and \
                     res_json['versions'].get(dependency_version) and \
                     res_json['versions'][dependency_version].get('keywords'):
                 d['keywords'] = res_json['versions'][dependency_version]['keywords']
 
+                # Version Date
+                dateTime = res_json['time'][dependency_version]
+
+                # Convert time format e.g. 2017-02-16T20:43:07.414Z -> 2017-02-16 20:43:07
+                date = dateTime.split('T')[0]
```

Comment:
https://momentjs.com is perhaps a better way to handle date formatting. Especially since you're doing this in more than one place.

Reply:
So we just pass 2017-02-16T20:43:07.414Z format of time and convert it in js side?